Light Vox Engine/Input/InputManager.cpp

```diff
@@ -0,0 +1,18 @@
+#include "InputManager.h"
+
+using namespace Input;
+
+void InputManager::OnMouseDown( WPARAM buttonState, int x, int y )
```

Comment:
Have you considered passing by reference here for less intensive parameter passing?